---
layout: page
title: Publications
---

# Featured publications

**Evans K. S.**, M. H. van Wijk, P. T. McGrath, E. C. Andersen, and M. G. Sterken, 2021 [From QTL to gene: C. elegans facilitates discoveries of the genetic mechanisms underlying natural variation](https://doi.org/10.1016/j.tig.2021.06.005). Trends Genet. 

*Although many studies have examined quantitative trait variation across many species, only a small number of genes and thereby molecular mechanisms have been discovered. Without these data, we can only speculate about evolutionary processes that underlie trait variation. Here, we review how quantitative and molecular genetics in the nematode Caenorhabditis elegans led to the discovery and validation of 37 quantitative trait genes over the past 15 years. Using these data, we can start to make inferences about evolution from these quantitative trait genes, including the roles that coding versus noncoding variation, gene family expansion, common versus rare variants, pleiotropy, and epistasis play in trait variation across this species.*

**Evans K. S.**, S. Zdraljevic, L. Stevens, K. Collins, R. E. Tanny, et al., 2020 [Natural variation in the sequestosome-related gene, sqst-5, underlies zinc homeostasis in Caenorhabditis elegans](https://doi.org/10.1371/journal.pgen.1008986). PLoS Genet. 16: e1008986. 

*Zinc is an essential trace element that acts as a co-factor for many enzymes and transcription factors required for cellular growth and development. Altering intracellular zinc levels can produce dramatic effects ranging from cell proliferation to cell death. To avoid such fates, cells have evolved mechanisms to handle both an excess and a deficiency of zinc. Zinc homeostasis is largely maintained via zinc transporters, permeable channels, and other zinc-binding proteins. Variation in these proteins might affect their ability to interact with zinc, leading to either increased sensitivity or resistance to natural zinc fluctuations in the environment. We can leverage the power of the roundworm nematode Caenorhabditis elegans as a tractable metazoan model for quantitative genetics to identify genes that could underlie variation in responses to zinc. We found that the laboratory-adapted strain (N2) is resistant and a natural isolate from Hawaii (CB4856) is sensitive to micromolar amounts of exogenous zinc supplementation. Using a panel of recombinant inbred lines, we identified two large-effect quantitative trait loci (QTL) on the left arm of chromosome III and the center of chromosome V that are associated with zinc responses. We validated and refined both QTL using near-isogenic lines (NILs) and identified a naturally occurring deletion in sqst-5, a sequestosome-related gene, that is associated with resistance to high exogenous zinc. We found that this deletion is relatively common across strains within the species and that variation in sqst-5 is associated with zinc resistance. Our results offer a possible mechanism for how organisms can respond to naturally high levels of zinc in the environment and how zinc homeostasis varies among individuals.*

**Evans K. S.**, J. Wit, L. Stevens, S. R. Hahnel, B. Rodriguez, et al., 2021 [Two novel loci underlie natural differences in Caenorhabditis elegans abamectin responses](https://doi.org/10.1371/journal.ppat.1009297). PLoS Pathog. 17: e1009297. 

*Parasitic nematodes cause a massive worldwide burden on human health along with a loss of livestock and agriculture productivity. Anthelmintics have been widely successful in treating parasitic nematodes. However, resistance is increasing, and little is known about the molecular and genetic causes of resistance for most of these drugs. The free-living roundworm Caenorhabditis elegans provides a tractable model to identify genes that underlie resistance. Unlike parasitic nematodes, C. elegans is easy to maintain in the laboratory, has a complete and well annotated genome, and has many genetic tools. Using a combination of wild isolates and a panel of recombinant inbred lines constructed from crosses of two genetically and phenotypically divergent strains, we identified three genomic regions on chromosome V that underlie natural differences in response to the macrocyclic lactone (ML) abamectin. One locus was identified previously and encodes an alpha subunit of a glutamate-gated chloride channel (glc-1). Here, we validate and narrow two novel loci using near-isogenic lines. Additionally, we generate a list of prioritized candidate genes identified in C. elegans and in the parasite Haemonchus contortus by comparison of ML resistance loci. These genes could represent previously unidentified resistance genes shared across nematode species and should be evaluated in the future. Our work highlights the advantages of using C. elegans as a model to better understand ML resistance in parasitic nematodes.*

---

# All publications

**Evans K. S.**, M. H. van Wijk, P. T. McGrath, E. C. Andersen, and M. G. Sterken, 2021 [From QTL to gene: C. elegans facilitates discoveries of the genetic mechanisms underlying natural variation](https://doi.org/10.1016/j.tig.2021.06.005). Trends Genet. 

**Evans K. S.**, J. Wit, L. Stevens, S. R. Hahnel, B. Rodriguez, et al., 2021 [Two novel loci underlie natural differences in Caenorhabditis elegans abamectin responses](https://doi.org/10.1371/journal.ppat.1009297). PLoS Pathog. 17: e1009297. 

**Evans K. S.**, S. Zdraljevic, L. Stevens, K. Collins, R. E. Tanny, et al., 2020 [Natural variation in the sequestosome-related gene, sqst-5, underlies zinc homeostasis in Caenorhabditis elegans](https://doi.org/10.1371/journal.pgen.1008986). PLoS Genet. 16: e1008986. 

**Evans K. S.**, and E. C. Andersen, 2020b [The cadmium-responsive gene, cdr-6, does not influence Caenorhabditis elegans responses to exogenous zinc](https://www.micropublication.org/journals/biology/micropub.biology.000305/). microPublication Biology 2020.

**Evans K. S.**, and E. C. Andersen, 2020a [The Gene scb-1 Underlies Variation in Caenorhabditis elegans Chemotherapeutic Responses](https://doi.org/10.1534/g3.120.401310). G3. 

Crombie T. A., S. Zdraljevic, D. E. Cook, R. E. Tanny, S. C. Brady, **K. S. Evans**, et al., 2019 [Deep sampling of Hawaiian Caenorhabditis elegans reveals high genetic diversity and admixture with global populations](https://doi.org/10.7554/eLife.50465). Elife 8: e50465. 

Kim C., J. Kim, S. Kim, D. E. Cook, **K. S. Evans**, et al., 2019 [Long-read sequencing reveals intra-species tolerance of substantial structural variations and new subtelomere formation in C. elegans](https://doi.org/10.1101/gr.246082.118). Genome Res. 29: 1023–1035. 

**Evans K. S.**, S. C. Brady, J. S. Bloom, R. E. Tanny, D. E. Cook, et al., 2018 [Shared Genomic Regions Underlie Natural Variation in Diverse Toxin Responses](https://doi.org/10.1534/genetics.118.301311). Genetics. 

Currie S. L., J. J. Doane, **K. S. Evans**, N. Bhachech, B. J. Madison, et al., 2017 [ETV4 and AP1 transcription factors form multivalent interactions with three sites on the MED25 activator-interacting domain](https://doi.org/10.1016/j.jmb.2017.06.024). J. Mol. Biol. 

**Evans K. S.**, Y. Zhao, S. C. Brady, L. Long, P. T. McGrath, et al., 2017 [Correlations of Genotype with Climate Parameters Suggest Caenorhabditis elegans Niche Adaptations](https://doi.org/10.1534/g3.116.035162). G3 7: 289–298. 